rest-gateway/pom.xml

```diff
@@ -69,6 +69,99 @@
     <build>
         <finalName>spline-rest-server-${project.version}</finalName>
         <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>exec-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>gen-swagger-consumer-json</id>
+                        <phase>process-classes</phase>
+                        <goals>
+                        <goal>java</goal>
+                        </goals>
+                        <configuration>
+                            <includePluginDependencies>true</includePluginDependencies>
+                            <executableDependency>
+                                <groupId>za.co.absa.spline</groupId>
+                                <artifactId>rest-api-doc-generator</artifactId>
+                            </executableDependency>
+                            <mainClass>za.co.absa.spline.swagger.SwaggerDocGenCLI</mainClass>
+                            <arguments>
+                                <argument>-o</argument>
+                                <argument>${project.basedir}/src/main/resources/static/consumerSwagger.json</argument>
```

Comment:
What about deduplicating `${project.basedir}/src/main/resources/static` part of the path with a custom maven property, say `${swagger.output.path}`?

Reply:
Done